partiql-tests-data/eval/query/order-by.ion

```diff
@@ -232,6 +232,122 @@ simple::[
       output:[{'supplierId': 10, n: 3}, {'supplierId': 11, 'n':2}]
     }
   },
+  // should order by aggregation function directly
+  {
+    name:"order by aggregation function",
+    statement:"SELECT supplierId, MAX(price) FROM products GROUP BY supplierId ORDER BY MAX(price) DESC",
+    assert:{
+      evalMode:[EvalModeCoerce, EvalModeError],
+      result:EvaluationSuccess,
+      output:[{'supplierId': 10, 'MAX': 15.0}, {'supplierId': 11, 'MAX': 15.0}]
+    }
+  },
+]
+
+alias::[
+  // ORDER BY alias from SELECT
+  {
+    name:"order by select alias",
+    statement:"SELECT productId AS pid, price AS cost FROM products ORDER BY pid",
```

Comment:
For better demonstration purposes, it's better to make the order by specification for `pid` `DESC`. The `products` table is already sorted in ascending order by `productId`.